src/views/pages/index.jsx

```diff
@@ -1,18 +1,118 @@
+        CartController.turnInLostEquipment(equipAddr);
+        this.cancelItemLost();
+    }
+
+    lostItem() {
+        this.setState({ lostItemModalActive: true });
+    }
+
+    cancelItemLost() {
+        this.setState({
+            inputModalInput: '',
+            inputModalIsInvalid: false,
+            lostItemModalActive: false
+        });
+    }
+
+    update(e) {
+        let equipAddr = e.target.value;
+        if (equipAddr.length > 0) {
+            try {
+                readAddress(equipAddr);
+                this.setState({
+                    inputModalIsInvalid: undefined,
+                    inputModalAcceptDisabled: false
+                });
+            } catch (err) {
+                this.setState({
+                    inputModalIsInvalid: true,
+                    inputModalAcceptDisabled: true
+                });
+            }
+        }
+        this.setState({ inputModalInput: equipAddr });
+    }
+
     render() {
         return (
             <div id='index'>
+                <InputModal
+                    message='Please enter the item or model address located under the barcode.'
+                    active={this.state.lostItemModalActive}
+                    onAccept={this.acceptItemLostAddr.bind(this)}
+                    onCancel={this.cancelItemLost.bind(this)}
+                    update={this.update.bind(this)}
+                    errorMessage='Equipment address is incorrect. Please double check adress!'
+                    acceptText='Enter'
+                    textHidden={false}
+                    input={this.state.inputModalInput}
+                    invalid={this.state.inputModalIsInvalid}
+                    acceptDisabled={this.state.inputModalAcceptDisabled}
+                />
                 <div id='links'>
-                  <button id='view-models' onClick={IndexController.getModels}>View all models</button><br/>
-                  <button id='view-items' onClick={IndexController.getItems}>View all items</button><br/>
-                  <button onClick={IndexController.getOverdueItems}>Overdue Items</button><br/>
-                  <button id='view-students' onClick={IndexController.getStudents}>View All Students</button><br/>
-                  <Link to='/students/upload'>Upload Student List</Link>
+                    <div id='view-models' onClick={IndexController.getModels}>
+                        <span>
+                            <img src="../assets/images/list.svg"/><br/>
+                            Models
+                        </span>
+                    </div>
+                    <div id='view-items' onClick={IndexController.getItems}>
+                        <span>
+                            <img src="../assets/images/list.svg"/><br/>
+                            Items
+                        </span>
+                    </div>
+                    <div id='new-model' onClick={() => hashHistory.push(`/models/new`)}>
```

Comment:
Can we keep the onClick's consistent and have them all use the IndexController? I know OmnibarController has a `leavePage(path)` function.